infra/terraform/modules/kubernetes_cluster/bin/upgrade_if_version_differs.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -eu -o pipefail
+
+DOMAIN=$1
+DECLARED_VERSION=$2
+
+# make sure we're looking at the right cluster
+kubectl config use-context $DOMAIN
```

Comment:
Nice one :D

Reply:
Actually that's not used anymore either - the kops YML files are regenerated and updated on every run, and `kops rolling-update` is always called, as it's a no-op if no update is required.